src/component/handlers/edit/editOnKeyDown.js

```diff
@@ -146,6 +148,19 @@ function editOnKeyDown(e: SyntheticKeyboardEvent): void {
   // we don't want to insert a character following the keydown.
   e.preventDefault();
 
+  if(isIOS && command === 'backspace') {
+    // On iOS 'backspace' acts as 'backspace-word'
+    // starting the 23rd continous event
+    if(continuousBackspaceCount++ >= 22) {
+      command = 'backspace-word';
+    }
+    setTimeout((previousCount) => {
+      if(continuousBackspaceCount === previousCount) {
+        continuousBackspaceCount = 0;
+      }
+    }, 400, continuousBackspaceCount);
```

Comment:
I don't think we should queue up timeouts on every backspace. This seems like it could lead to some timing issues -- none of these timeouts will be cleared.

This seems like a classic use case for `debounce`.

```javascript
const tryToResetBackspaceCount = debounce(() => {
  continuousBackspaceCount = 0;
}, MSEC);

// Later...

if (isIOS && command === 'backspace') {
  if (continuousBackspaceCount++ >= 22) {
    command = 'backspace-word';
  }
  tryToResetBackspaceCount();
}
```

This way, on every `backspace` command, we try to reset. If the command occurs again within `MSEC`, under the hood the timeout will be cleared and the counter will not be reset. If no command occurs within `MSEC`, the counter will be reset.